src/Models/UserIdentityModel.php

```diff
@@ -165,7 +165,7 @@ public function revokeAllAccessTokens($userId)
             ->delete();
     }
 
-    public function fake(Generator &$faker)
+    public function fake(Generator &$faker): array
```

Comment:
Why `UserIdentity`?
It is an array.

Reply:
Right, the return statement will need to be updated as well:
```php
return new UserIdentity([
```
It isn't a requirement, but I generally assume that a faked model result will match its return type.

Reply:
>IMO fake() methods should always return $returnType

Do you mean the user guide shows not good sample code?
https://codeigniter4.github.io/CodeIgniter4/testing/fabricator.html#defining-formatters

Reply:
Fixed.

Reply:
The User Guide examples look fine to me. The one `UserModel` example that returns an array has no defined `$returnType` property which means it defaults to `array`. 

Maybe this is just an opinion and not even a best practice, but I think of `fake()`/`make()` as "create an example of this model's output using fake data". Defining this in the Model itself (instead of a stand-alone Fabricator) adds to that expectation.